### DISCUSSION TITLE
Matchers conditions that succeed requests?

### DISCUSSION THREAD

**Discussion by JaneX8** [projectdiscovery/nuclei#2936] [Q&A] [answered]
I read the documentations about `matchers-condition: ` and `condition:`. The first is helpful when multiple matchers are used in one request. The second is helpful when multiple criteria are used within one matcher (thus: one request).

I'm searching for a way to have matcher conditions succeed requests. For example I use two requests to check if a DNS record exists on the `{{RDN}}` and a second request on the `{{FQDN}}`. Now I only want to "match" when BOTH requests report non-existence of the record.

What do I use for that?

**Comment by ehsandeep** [comment, marked as answer]
@JaneX8 something like that would be supported with https://github.com/projectdiscovery/nuclei/issues/2534
